python/ray/autoscaler/kuberay/kuberay-autoscaler.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/ray-operator/config/default/kustomization.yaml b/ray-operator/config/default/kustomization.yaml
+index 333106d..fdf62d6 100644
+--- a/ray-operator/config/default/kustomization.yaml
++++ b/ray-operator/config/default/kustomization.yaml
+@@ -21,6 +21,6 @@ bases:
+ 
+ images:
+ - name: kuberay/operator
+-  newName: kuberay/operator
+-  newTag: nightly
++  newName: rayproject/kuberay-operator
++  newTag: latest
```

Comment:
how do we like to maintain the image in the future?

Reply:
Ideally, we have versioned releases of the image in `kuberay/operator`, either by doing releases with a version number, or just by git commit hash if we want to keep it low overhead that would be fine too. Then we can pin them here and update them whenever there is a new release :)